docs/modules/ROOT/pages/documentation/payara-server/server-configuration/http/protocols.adoc

```diff
@@ -13,3 +13,604 @@ commands. The protocol configurations can be found in the admin console under
 
 image::http/protocol-config.png[Configuration Page Location]
 
+== Standard Options
+
+The following standard fields can be configured for protocols:
```

Comment:
This sentence is confusing. It's better to state: "The following are the main configuration settings for network listener protocols"